Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -21,7 +21,7 @@ First the Prometheus operator needs to be started in the cluster so it can watch
 A full explanation can be found in the [Prometheus operator repository on GitHub](https://github.com/prometheus-operator/prometheus-operator), but the quick instructions can be found here:
 
 ```console
-kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
+kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.81.0/bundle.yaml
```

Comment:
```suggestion
kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.82.0/bundle.yaml
```
?

Reply:
Version 82 was released just 4 days ago, so I preferred to work with version 81 for now.
@subhamkrai Do you prefer that we switch to version 82 instead?
https://github.com/prometheus-operator/prometheus-operator/tags

Reply:
The prometheus version is pretty low risk, since we're basically just updated it in our docs. If we could just manually verify that the user can basically configure prometheus according to our doc (up through line 68), that is likely sufficient. So I'd suggest just go ahead with the latest 0.82 even if it just came out.

Even better if you want to verify the entire doc, but that's really a separate work item since this doc was created so long ago and may be outdated.

Reply:
@travisn @subhamkrai I’ve updated the version to 0.82. I’ll also manually test the steps in the doc on a local Minikube setup and share the results here: https://rook.io/docs/rook/latest-release/Storage-Configuration/Monitoring/ceph-monitoring/#prometheus-operator

Reply:
@OdedViner link you shared is Rook official link and I don't we'll able to see your local test on docs updates.

Also if you just update the PR title and commit tittle to 0.82 instead of 0.81

Reply:
https://docs.google.com/document/d/1lqab3yqlmvLlI0gMvLDOjsPLnUlsMnHWCE5xLWZSHAo/edit?tab=t.0